polkadot/node/subsystem-bench/src/lib/configuration.rs

```diff
@@ -241,3 +249,183 @@ pub fn random_latency(maybe_peer_latency: Option<&PeerLatency>) -> usize {
 		})
 		.unwrap_or(0.0) as usize
 }
+
+pub struct TestConfigurationBuilder {
+	/// Number of validators
```

Comment:
I think we don't need to repeat the documentation here

Reply:
comments removed.